Removed -y

outzhu opened the pull request:
I thought the changes made from the previous PR would be committed to my Forked.

outzhu committed:
Update Dockerfile

Signed-off-by: Ada-Codina <adacodina@gmail.com>

oxr463 commented:
Now we have to fix these issues to get the pipeline to succeed, https://github.com/aad-for-linux/docker/runs/4093493128?check_suite_focus=true#step:3:17

outzhu committed:
4 errors package

libjansson-dev \
libpam0g-dev \
#these were found in https://pkgs.alpinelinux.org/packages
quilt \
libsds

outzhu commented:
Will run in my forked until it's fixed.
